sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_operations/_patch.py

```diff
@@ -7,299 +7,9 @@
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
 from typing import List
-from ._operations import (
-    ConversationAnalysisClientOperationsMixin as ConversationAnalysisClientOperationsMixinGenerated,
-)
 
 
-ConversationAnalysisClientOperationsMixinGenerated.analyze_conversation.__doc__ = """Analyzes the input
```

Comment:
Why was all this removed? Thus far in the review, it begs the question of whether this breaks compat elsewhere? Or was all this done by an updated generator somewhere else?

Reply:
I had originally copied over all of the generated docstrings into this file so I could manually add the REST API links (this was before the swagger transform was able to do this for us).

Reply:
I assumed as much initially, but didn't see any links which raised the question.

Reply:
you're right, looks like I missed the link for begin_conversation_analysis. :/ To be fair, I am removing more in this file than our generator will make up for, but this was feedback from Johan to link out to the docs rather than manually including all the polymorphic requests/responses in the docstrings...